packages/rocketchat-videobridge/client/actionLink.js

```diff
@@ -4,7 +4,7 @@ RocketChat.actionLinks.register('joinJitsiCall', function(/*message, params*/) {
 
 		let room = RocketChat.models.Rooms.findOne({_id: rid});
 		let currentTime = new Date().getTime();
-		let jitsiTimeout = new Date(room.jitsiTimeout).getTime() || currentTime;
+		let jitsiTimeout = new Date((room && room.jitsiTimeout) || currentTime).getTime();
```

Comment:
not really @geekgonecrazy .. javascript return the value from the last "parameter" and not a boolean of the sentence (like for example, PHP does)..
so:
```
var obj = { foo: 'bar' };
console.log( obj && obj.foo ); // prints bar
```